src/Bicep.Local.Extension/Rpc/GrpcChannelHelper.cs

```diff
@@ -48,7 +48,7 @@ public static GrpcChannel CreateChannel(string socketPath)
         };
 
         // The URL is not used, but it must be a valid URI.
-        return GrpcChannel.ForAddress("invalid://unused", new GrpcChannelOptions
```

Comment:
I had to change this back because it caused a test failure - this method requires a known scheme (e.g. `http`)